entity-service/build.gradle.kts

```diff
@@ -34,6 +36,9 @@ tasks.bootRun {
 
 jib.from.image = project.ext["jibFromImage"].toString()
 jib.to.image = "stellio/stellio-entity-service"
-jib.container.jvmFlags = listOf(project.ext["jibContainerJvmFlag"].toString())
+jib.container.entrypoint = listOf("/bin/sh")
+jib.container.args = listOf("-c", "chmod +x /database/wait-for-neo4j.sh && " +
+    "/database/wait-for-neo4j.sh neo4j:7687 -t 40 -- echo \"Connected to Neo4j\" && " +
```

Comment:
why don't you do `wait-for-neo4j.sh ... -- java ...`?

Reply:
it is not possible to add the script with the correct permissions by using `extraDirectories.permissions`?

Reply:
> it is not possible to add the script with the correct permissions by using `extraDirectories.permissions`?

Yes

Reply:
> why don't you do `wait-for-neo4j.sh ... -- java ...`?

You mean ?
```
"/database/wait-for-neo4j.sh neo4j:7687 -t 40 -- echo \"Connected to Neo4j\" -- " +
    "java " + project.ext["jibContainerJvmFlag"].toString() + " -cp /app/resources:/app/classes:/app/libs/* " + mainClass)
```

Reply:
Or ?

```
"/database/wait-for-neo4j.sh neo4j:7687 -t 40 -- " +
    "java " + project.ext["jibContainerJvmFlag"].toString() + " -cp /app/resources:/app/classes:/app/libs/* " + mainClass
```

Reply:
the second one

Reply:
> > it is not possible to add the script with the correct permissions by using `extraDirectories.permissions`?
> 
> Yes

why it is not possible?

Reply:
sorry i meant it's possible